benchmarks/math/triangulate.gd

```diff
@@ -0,0 +1,28 @@
+extends Benchmark
+
+const RADIUS := 1.0
+const NUM_SIDES := 10_000
+const POSITION := Vector2(0.0, 0.0)
+
+# Generate a polygon circle of a certain `radius` with
+# a certain `num_side` and `position` as the center point.
+func generate_circle_polygon(radius: float,
```

Comment:
Based on this [answer](https://ask.godotengine.org/81776/how-to-shape-polygon2d-into-a-circle?show=140975#a140975) from Godot's Q&A website.